msteams-platform/bots/how-to/conversations/subscribe-to-conversation-events.md

```diff
@@ -1465,3 +1465,7 @@ It's important to know that new events can be added anytime in the future and yo
 
 > [!div class="nextstepaction"]
 > [Send proactive messages](~/bots/how-to/conversations/send-proactive-messages.md)
+
+## See also
+
+* [Triggers in Bot Framework Composer](/composer/concept-events-and-triggers)
```

Comment:
@ashishguptaiitb, 

See also has only link and we won't add bullet for one item.